content/02.main-text.md

```diff
@@ -220,6 +220,10 @@ manubot cite --render --format=markdown \
   pmid:29618526 doi:10.1038/550143a
 ```
 
+We are also [exploring](https://github.com/manubot/manubot/pull/99 "Create a pandoc-manubot-cite filter for pandoc") whether we can implement citation-by-identifier as a Pandoc filter.
```

Comment:
Do we want the link in the main text or the response?  I'm not sure the likelihood that this PR will be completed.  If there is a reasonable chance, we'll go in a different direction, we could keep the sentiment of this paragraph but be slightly less specific.

I like the paragraph overall and it's contribution to the future vision.

Reply:
I am pretty convinced a pandoc filter is the right move at this point. At this point, the filter in https://github.com/manubot/manubot/pull/99 is functional, so the proof of concept is there and I imagine we'll debut the filter around the time this paper is published.

Reply:
👍